cloud/blockstore/libs/endpoints/endpoint_manager.cpp

```diff
@@ -551,6 +551,10 @@ class TEndpointManager final
         });
     }
 
+    TFuture<NProto::TStartEndpointResponse> RestoreSingleEndpointForTesting(
```

Comment:
generally not a good idea to add interface api just for test ... why not call `manager->RestoreEndpoints()` directly